collector/README.md

```diff
@@ -516,23 +516,28 @@ Codegen diff is currently only implemented for runtime benchmarks.
 
 ## Binary size statistics
 You can use the `binary_stats` command to display size statistics (section and symbol sizes) of
-binary artifacts (executables, libraries) of selected compile benchmarks.
-
-```
-./target/release/collector binary_stats `<rustc>` --include <benchmark name> \
-    [--profile <Debug|Opt>] \
-    [--backend <Llvm|Cranelift>]
-```
-
-You can also compare (diff) the size statistics between two compilers:
-```
-./target/release/collector binary_stats `<rustc>` --include <benchmark name> --rustc2 <rustc2>
-```
-or between two codegen backends:
-```
-./target/release/collector binary_stats `<rustc>` --include <benchmark name> --rustc2 <rustc>
-    --backend <Llvm|Cranelift> --backend2 <Llvm|Cranelift>
-```
+binary artifacts (executables, libraries). You can compare the binary statistics of:
+
+- Selected compile benchmarks:
+  ```bash
+  ./target/release/collector binary_stats compile `<rustc>` --include <benchmark name> \
+      [--profile <Debug|Opt>] \
+      [--backend <Llvm|Cranelift>]
+  ```
+  
+  You can also compare (diff) the size statistics between two compilers:
+  ```bash
+  ./target/release/collector binary_stats compile `<rustc>` --include <benchmark name> --rustc2 <rustc2>
+  ```
+  or between two codegen backends:
+  ```bash
+  ./target/release/collector binary_stats compile `<rustc>` --include <benchmark name> --rustc2 <rustc>
+      --backend <Llvm|Cranelift> --backend2 <Llvm|Cranelift>
+  ```
+- Arbitrary binary artifacts on disk:
+  ```bash
+  ./target/release/collector binary_stats local `<artifact-path>` [<artifact-to-compare-to>]
```

Comment:
The "artifact" I recognize is the version of `rustc`. I wonder if using "binary artifacts" to refer to other things would cause confusion? What about just telling the user that it is a binary executable or library directly

Reply:
Well, naming is hard :) Feel free to send a PR that clarifies it to binary/library, if you want.